### PR TITLE
Base database model

### PR DESCRIPTION
**Pull request by mdmoralesbarreda**
Created all migrations with raw SQL

### CODE REVIEW

**Review by adriangromero — APPROVED** [review]
LGTM

**Comment by juanfumero** [comment]
aprovate pull request

File: migrations/20190928142938-coordinates.js [hunk 1]
```diff
@@ -15,31 +15,22 @@ exports.setup = function(options, seedLink) {
...
+  'CREATE TABLE coordinates (' +
+  'coordinate_id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,'+
+  'longitude DECIMAL(10,6) SIGNED NOT NULL,'+
+  'latitude DECIMAL(10,6) SIGNED NOT NULL,'+
+  'time DATETIME NOT NULL,'+
+  'elevation DECIMAL (7,2) SIGNED,'+
+  'created_at DATETIME DEFAULT CURRENT_TIMESTAMP'+
+  ')';
+
+  db.runSql(createCoordinatesSentence, function(err) {
```
Comment (imedgar) [inline migrations/20190928142938-coordinates.js:29]: We should add '_+ as an ESLint rule. Anyway we can change it later

File: migrations/20191006170617-groups.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+var dbm;
+var type;
+var seed;
```
Comment (imedgar) [inline migrations/20191006170617-groups.js:5]: var -> let

File: migrations/20191006170617-groups.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+exports.up = function(db, callback) {
+
+  const createGroupsSentence =
+  'CREATE TABLE groups (' +
+  'group_id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,'+
+  'name VARCHAR(255),'+
+  'status TINYINT(1) DEFAULT 1,'+
+  'created_at DATETIME DEFAULT CURRENT_TIMESTAMP,'+
+  'updates_at DATETIME ON UPDATE CURRENT_TIMESTAMP,'+
+  'deleted_at DATETIME'+
```
Comment (imedgar) [inline migrations/20191006170617-groups.js:26]: same here

File: migrations/20191006170617-groups.js [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+  db.runSql(createGroupsSentence, function(err) {
+    if (err) return callback(err);
+    return callback();
+  });
+
+};
+
+exports.down = function(db, callback) {
+  db.dropTable('groups',callback);
```
Comment (imedgar) [inline migrations/20191006170617-groups.js:37]: ,_callback

File: migrations/20191006184624-users.js [hunk 5]
```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+var dbm;
+var type;
+var seed;
```
Comment (imedgar) [inline migrations/20191006184624-users.js:5]: let

**Review by imedgar — APPROVED** [review]
minor comments, LGTM

**Pull request merged**